src/old/modules/postUpdation/components/TextPostUpdation.tsx

```diff
@@ -134,8 +134,8 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
   };
 
   const handlePublishClick = async () => {
-    const response = await updatePost(updatedPost);
-    history.push(`/posts/${response.data.id}`);
+    await updatePost(updatedPost);
```

Comment:
All new changes should be done in new files structure (in this case: src/views/postUpdation/TextPostUpdation.tsx)
#323